### PR TITLE
[oneinch] add oneinch_fusion_executors and fix naming in oneinch_contract_addresses

### PR DESCRIPTION
**Pull request by grkhr**
…ract_addresses

Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Comment by grkhr** [comment]
ready for review

File: models/oneinch/oneinch_fusion_executors.sql [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    ) as t(resolver_address, resolver_name, kyc)
+)
+
+, traces as (
+    select 
+        tx_hash
+        , "from" as resolver_address
+        , substring(input, 49, 20) as resolver_executor
+        , cast(bytea2numeric(substring(input, 5, 32)) as double) as chain_id
+    from ethereum.traces
```
Comment (jeff-dude) [inline models/oneinch/oneinch_fusion_executors.sql:59]: please apply jinja syntax on the source here

File: models/oneinch/oneinch_fusion_executors.sql [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        ('0x7c7047337995c338c1682f12bc38d4e4108309bb', 'Resolver 0x7c70..09bb', false)
+    ) as t(resolver_address, resolver_name, kyc)
+)
+
+, traces as (
+    select 
+        tx_hash
+        , "from" as resolver_address
+        , substring(input, 49, 20) as resolver_executor
+        , cast(bytea2numeric(substring(input, 5, 32)) as double) as chain_id
```
Comment (jeff-dude) [inline models/oneinch/oneinch_fusion_executors.sql:58]: we have a newer function in spellbook, if interested to use here (it runs on spark fine):
https://github.com/duneanalytics/spellbook/blob/main/macros/public/bytea2numeric_v3.sql

File: models/oneinch/oneinch_schema.yml [hunk 3]
```diff
@@ -27,5 +27,34 @@ models:
...
+  - name: oneinch_fusion_executors
+    meta:
+      blockchain: ['ethereum','optimism','polygon','arbitrum','avalanche_c','gnosis','bnb','fantom']
+      sector: oneinch
+      contributors: [grkhr]
+    config:
+      tags: ['oneinch', 'metadata']
+    description: >
+        fusion resolvers and executors
+    tests:
```
Comment (jeff-dude) [inline models/oneinch/oneinch_schema.yml:42]: thanks for applying the tests 🙏

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/oneinch/oneinch_fusion_executors.sql (hunk 1 above)
Comment (grkhr) [reply models/oneinch/oneinch_fusion_executors.sql:59]: ahhh. sorry!

**Comment by grkhr** [comment]
@jeff-dude fixed!

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**